### ISSUE TITLE
Mbed setup.  Instructions please...

### ISSUE THREAD

**Issue by winneymj** [sethitow/mbed-pinetime#1]
Hi,
I am interested in building your repo but it does not seem to be a standard Mbed project setup and I am not sure how to build.
Please can you provide some more set instruction please.
Thanks
Mark

**Comment by sethitow** [comment]
Hi Mark,

Sorry I’m just getting to this. You are correct that this project does not currently use the Mbed CLI. To build it, you need a version of `make`. If you can tell me what OS you’re using, I can work on providing better instructions. 

I’m open to suggestions on ways to improve the build system to make it easier for others to get started. 

Seth

**Comment by winneymj** [comment]
Hi,
I kinda figured out that it will not work on windows.  Seems there is a
known problem with command line lengths passed to the compiler.
I managed to start a build by running "make build" command down in the
"app/peripheral_test" folder but it fails with "directory not found".
Googling this I found lots of problems with command line length of 32K
being exceeded which I believe is happening here.
A latest fix (which I am not sure you have) was to change the paths in the
.include_xxxx.txt file to relative pathing instead of absolute, which might
help.
Seems the change was only applied to building test files though so not sure
it will help.  Here is link to mbed issue:
https://github.com/ARMmbed/mbed-os/pull/6968/commits/3e7fd9e0f001ab3eb7e86c23465ef79003c47e8a


Anyhow, have you done a build in Ubuntu?  I have a windows Ubuntu WSL
installed, but still having problems building.
Will work on it a little more later.
Thanks for following up.  I closed the issue after I found the 32K window
bug.
Have a good new year
Mark.
